sync/class.jetpack-sync-module-protect.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+require dirname( __FILE__ ) . '/../modules/protect.php';
+
+/** 
+ * logs bruteprotect failed logins via sync
+ */
+class Jetpack_Sync_Module_Protect extends Jetpack_Sync_Module {
+	private $taxonomy_whitelist;
+	
+	function name() {
+		return "protect";
```

Comment:
Good call. I'll open a separate PR which fixes this for all the "name" function return values.